### PR TITLE
[script][equipmanager] Add matches for cold gear

### PR DESCRIPTION
**Pull request by Raykyn55**
```
[crossing-repair]>wear my bearskin.gloves

A brisk chill rushes through you as you wear some hiro bearskin gloves interlaced with strips of ice-veined leather.
Ravendrake drops his large sack in a large waste bucket.
Sightless Ravendrake goes out.
>;chat discipline helps a little

[crossing-repair: *** No match was found after 15 seconds, dumping info]
```
```
[crossing-repair]>remove my bearskin.gloves

A brisk chill leaves you as you remove some hiro bearskin gloves interlaced with strips of ice-veined leather.
You feel fully rested.
 * Intern ----- joins the adventure.

[crossing-repair: *** No match was found after 15 seconds, dumping info]
```
post edit:

```
>
[crossing-repair]>remove my bearskin.gloves
A brisk chill leaves you as you remove some hiro bearskin gloves interlaced with strips of ice-veined leather.
>
Kaldean effortlessly begins a quiet concerto on his copper zills, his eyes sparkling with pride at his hard-won skill.
>
[crossing-repair]>give my bearskin.gloves to Ylono
Kaldean puts his ticket in his rucksack.
```
```
[crossing-repair]>get my Ylono ticket
You get a Ylono repair ticket from inside your scavenger's pack.
>
[crossing-repair]>give my ticket to Ylono
You hand Ylono your ticket and are handed back some hiro bearskin gloves interlaced with strips of ice-veined leather.
>
[crossing-repair]>wear my bearskin.gloves
A brisk chill rushes through you as you wear some hiro bearskin gloves interlaced with strips of ice-veined leather.
>
Kaldean puts his mace in his chest harness.
>
[crossing-repair]>get my Ylono ticket
```

### CODE REVIEW

**Comment by Raykyn55** [comment]
Other pieces:
```
>wear cowl
A brisk chill rushes through you as you wear a hiro bearskin cowl layered with dark crimson wool and ice-veined leather.
>
A tingling on the back of your neck draws attention to itself by disappearing, making you believe the heat is off from your last break in.
>rem cowl
A brisk chill leaves you as you remove a hiro bearskin cowl layered with dark crimson wool and ice-veined leather.
```

**Comment by asechrest** [comment]
Thanks. Does the anchoring work when using the string version of bput matches?

File: equipmanager.lic [hunk 1]
```diff
@@ -272,7 +272,7 @@ class EquipmentManager
     todo.all? do |held_item|
       if info = worn_items.find { |item| item.short_regex =~ held_item }
         unload_weapon(info.short_name) if info.needs_unloading
-        stow_helper("wear my #{info.short_name}", info.short_name, 'You sling', 'You attach', 'You .* unload', 'You strap', 'You slide', 'You work your way into', 'You spin', '^You ', 'You carefully loop', 'slide effortlessly onto your', 'You slip')
+        stow_helper("wear my #{info.short_name}", info.short_name, 'You sling', 'You attach', 'You .* unload', 'You strap', 'You slide', 'You work your way into', 'You spin', '^You ', 'You carefully loop', 'slide effortlessly onto your', 'You slip', '^A brisk chill rushes through you')
```
Comment (asechrest) [inline equipmanager.lic:275]:
```suggestion
        stow_helper("wear my #{info.short_name}", info.short_name, 'You sling', 'You attach', 'You .* unload', 'You strap', 'You slide', 'You work your way into', 'You spin', '^You ', 'You carefully loop', 'slide effortlessly onto your', 'You slip', /^A brisk chill rushes through you/)
```

File: equipmanager.lic [hunk 2]
```diff
@@ -300,7 +300,7 @@ class EquipmentManager
     {
       worn: {
         verb: 'remove',
-        matches: [/^You .*#{item.short_regex}/, /^You (get|sling|pull|work|loosen|slide|remove|yank|unbuckle).*#{item.name}/, 'you tug', 'Remove what', "You aren't wearing that", 'slide themselves off of your', 'you manage to loosen'],
+        matches: [/^You .*#{item.short_regex}/, /^You (get|sling|pull|work|loosen|slide|remove|yank|unbuckle).*#{item.name}/, 'you tug', 'Remove what', "You aren't wearing that", 'slide themselves off of your', 'you manage to loosen', '^A brisk chill leaves you as you'],
```
Comment (asechrest) [inline equipmanager.lic:303]:
```suggestion
        matches: [/^You .*#{item.short_regex}/, /^You (get|sling|pull|work|loosen|slide|remove|yank|unbuckle).*#{item.name}/, 'you tug', 'Remove what', "You aren't wearing that", 'slide themselves off of your', 'you manage to loosen', /^A brisk chill leaves you as you/],
```

**Comment by Raykyn55** [comment]
> 

Im not sure i want to fix it until its broken. With no reported false matches that im aware of, the risk would be missed matches with it gone, no?

**Comment by asechrest** [comment]
> > 
> 
> Im not sure i want to fix it until its broken. With no reported false matches that im aware of, the risk would be missed matches with it gone, no?

Yeah, that's probably a fair point, and you're right, we haven't had trouble with false matches. Thanks for the PR, merging.

**Pull request merged**